sympy/concrete/tests/test_sums_products.py

```diff
@@ -255,6 +255,12 @@ def test_geometric_sums():
     #issue 9908:
     assert Sum(1/(n**3 - 1), (n, -oo, -2)).doit() == summation(1/(n**3 - 1), (n, -oo, -2))
 
+    #issue 11642:
+    assert Sum(0.5**n,(n, 1, oo)).doit() == 1.0
```

Comment:
I suspect this comes back as `1`, not `1.0`. I think it should come back as a Float: if the input is not exact, the output should not be exact. You can test the result by `assert result.is_Float` in addition to testing the value (`result == 1`).